### PR TITLE
Add instructions for Fedora

### PR DESCRIPTION
**Pull request by fabaff**

### CODE REVIEW

**Comment by jpillora** [comment]
How does dnf download chisel? I'm guessing someone owns the name and (hopefully) republishes the chisel binaries?

**Comment by fabaff** [comment]
> How does dnf download chisel? I'm guessing someone owns the name and (hopefully) republishes the chisel binaries?

I created and maintain a [RPM package](https://src.fedoraproject.org/rpms/chisel) of `chisel`, thus it's coming out of the Fedora Package Collection if one wants to install it with `dnf`.

**Comment by jpillora** [comment]
Oh I see, could you PR an update the CI (GitHub actions + goreleaser) to
release the rpm package?

**Comment by fabaff** [comment]
> Oh I see, could you PR an update the CI (GitHub actions + goreleaser) to release the rpm package?

The RPM package is created on the [Fedora Build System](https://koji.fedoraproject.org/koji/). To build the RPM package all dependencies are RPM packages as well. 

At the moment I don't think that's possible to do this on GitHub.

File: README.md [hunk 1]
```diff
@@ -37,6 +37,12 @@ See [the latest release](https://github.com/jpillora/chisel/releases/latest) or
 docker run --rm -it jpillora/chisel --help
 ```
 
+**Fedora**
+
```
Comment (jpillora) [inline README.md:41]: Maybe we can put a `*Managed by @fabaff*` on this line?

**Comment by jpillora** [comment]
Happy to merge, just want to mention its not managed by me

**Comment by jpillora** [comment]
P.s. homeassistant is awesome! I use it every day :)

**Comment by jpillora** [comment]
Merged :)

**Comment by Semulv** [comment]
> _No description provided._

**Pull request merged**